src/main/java/io/debezium/connector/yugabytedb/connection/HashPartition.java

```diff
@@ -0,0 +1,390 @@
+package io.debezium.connector.yugabytedb.connection;
+
+import com.google.common.base.Objects;
+import io.debezium.DebeziumException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.yb.cdc.CdcService;
+import org.yb.client.AsyncYBClient;
+import org.yb.client.Bytes;
+import org.yb.client.GetTabletListToPollForCDCResponse;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * Partition class to store a representation of a tablet with its ranges. Most of the logic for
+ * this class has been taken from
+ * <a href="https://github.com/yugabyte/yugabyte-db/blob/master/java/yb-client/src/main/java/org/yb/client/Bytes.java">Bytes.java</a><br><br>
+ *
+ * Part of the logic to compare the two partitions has been borrowed from
+ * <a href="https://github.com/yugabyte/yugabyte-db/blob/master/src/yb/util/slice.h#L311">slice.h</a>
+ * @author Vaibhav Kushwaha (vkushwaha@yugabyte.com)
+ */
+public class HashPartition implements Comparable<HashPartition> {
+	private final String tableId;
```

Comment:
Some issue with formatting of this file

Reply:
is the formatting correct?

Reply:
The formatting seems to be fine in my IDE, probably something off with the GH preview. Will do a double check once again.